tpl/_backend_sitecode.markdown

```diff
@@ -111,6 +111,27 @@ it’s available:
 The default implementation already handles this, and you only need to worry
 about this if you call `count()` manually.
 
+### `url(vars)`
+Get URL to send to the server; the `vars` parameter behaves as `count()`.
+
+Note that you may want to use `filter()` to exclude prerender requests and
+various other things.
+
+### `filter()`
```

Comment:
Currently this function is not exposed to the end user (`var filter = function() {...}`).

Exposing it would require minor changes to the code calling it

Reply:
Whoops, fixed it now 😅